rs/registry/canister/src/registry.rs

```diff
@@ -231,19 +233,19 @@ impl Registry {
             } as i32;
         }
 
-        let req = RegistryAtomicMutateRequest {
-            mutations,
-            preconditions: vec![],
-        };
-        self.changelog_insert(version, &req);
-
-        for mutation in req.mutations {
-            (*self.store.entry(mutation.key).or_default()).push_back(RegistryValue {
+        for mutation in &mutations {
```

Comment:
why does the order of inserting the entries into the store and calling changelog_insert need to change?

Reply:
Didn't see those, b/c they were all resolved when I got to the PR.  Thanks for explaining.